netebpfext/user/fwp_um.cpp

```diff
@@ -269,7 +272,7 @@ _fwp_engine::test_cgroup_inet4_connect(_In_ fwp_classify_parameters_t* parameter
 
     action = test_callout(
         FWPS_LAYER_ALE_CONNECT_REDIRECT_V4, FWPM_LAYER_ALE_CONNECT_REDIRECT_V4, EBPF_DEFAULT_SUBLAYER, incoming_value);
-    ebpf_assert(action == FWP_ACTION_PERMIT);
+    ebpf_assert(action == FWP_ACTION_PERMIT || !fault_injection_enabled);
```

Comment:
Should this be `ebpf_assert(action == FWP_ACTION_PERMIT || fault_injection_enabled);` ?